x-pack/plugins/profiling/common/callercallee.ts

```diff
@@ -45,7 +45,7 @@ export function createCallerCalleeIntermediateNode(
   };
 }
 
-interface relevantTrace {
+interface RelevantTrace {
```

Comment:
What is the rule here ? I'd expect unexported interfaces to begin with lowercase (but that may be the Go linter only 😆 )

Reply:
Yeah, that's a Go thing then hehe. Interfaces/types should all be PascalCase.